pkg/apis/servicecatalog/validation/broker_test.go

```diff
@@ -208,12 +229,56 @@ func TestValidateServiceBroker(t *testing.T) {
 					Name: "test-broker",
 				},
 				Spec: servicecatalog.ServiceBrokerSpec{
-					URL:      "http://example.com",
-					CABundle: []byte("fake CABundle"),
+					URL:            "http://example.com",
+					CABundle:       []byte("fake CABundle"),
+					RelistBehavior: "Duration",
+					RelistDuration: &metav1.Duration{15 & time.Minute},
+				},
+			},
+			valid: true,
+		},
+		{
+			name: "valid broker - manual behavior with RelistDuration",
+			broker: &servicecatalog.ServiceBroker{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-broker",
+				},
+				Spec: servicecatalog.ServiceBrokerSpec{
+					URL:            "http://example.com",
+					RelistBehavior: "Manual",
+					RelistDuration: &metav1.Duration{15 & time.Minute},
+				},
+			},
+			valid: true,
+		},
+		{
+			name: "valid broker - manual behavior without RelistDuration",
+			broker: &servicecatalog.ServiceBroker{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-broker",
+				},
+				Spec: servicecatalog.ServiceBrokerSpec{
+					URL:            "http://example.com",
+					RelistBehavior: "Manual",
+					RelistDuration: nil,
 				},
 			},
 			valid: true,
 		},
+		{
+			name: "invalid broker - duration behavior without duration",
+			broker: &servicecatalog.ServiceBroker{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-broker",
+				},
+				Spec: servicecatalog.ServiceBrokerSpec{
+					URL:            "http://example.com",
+					RelistBehavior: "Duration",
+					RelistDuration: nil,
+				},
+			},
+			valid: false,
+		},
```

Comment:
We also need tests to verify the behavior when RelistBehavior is invalid and empty.